src/components/AccountCard.tsx

```diff
@@ -42,56 +43,60 @@ const AccountCard = ({
     getTagDerivationMode(currency as CryptoCurrency, account.derivationMode);
 
   return (
-    <Card
+    <TouchableOpacity
+      activeOpacity={0.6}
       onPress={!disabled ? onPress : undefined}
```

Comment:
I think here since it's now a TouchableOpacity you should simply use the `disabled` prop https://reactnative.dev/docs/touchablewithoutfeedback#disabled